cmd/kola/options.go

```diff
@@ -27,6 +29,11 @@ var (
 		"amd64-usr": sdk.BuildRoot() + "/images/amd64-usr/latest/coreos_production_image.bin",
 		"arm64-usr": sdk.BuildRoot() + "/images/arm64-usr/latest/coreos_developer_image.bin",
 	}
+
+	kolaDefaultBIOS = map[string]string{
+		"amd64-usr": "",
```

Comment:
Use `bios.bin` and then this flag can be used for both. And then it is possible to do kola on x86_64 uefi too :)